controllers/fenceagentsremediation_controller.go

```diff
@@ -202,6 +214,7 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		cmd := append([]string{far.Spec.Agent}, faParams...)
 		// The Fence Agent is excutable and the parameters structure are valid, but we don't check their values
 		r.Log.Info("Execute the fence agent", "Fence Agent", far.Spec.Agent, "Node Name", req.Name)
+		commonEvents.NormalEvent(r.Recorder, node, v1alpha1.EventReasonFenceAgentExecuted, v1alpha1.EventMessageFenceAgentExecuted)
```

Comment:
the fence angent was not executed yet a this point.  a message like "ExecuteFenceAgent", like the log above

Reply:
The idea was to send an event for every time the fence agent was executed